src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -221,12 +225,17 @@ export class IeObjectsController {
 	public async getSimilar(
```

Comment:
Now that we figured out which endpoint is which, lets add some documentation to the swagger file:
```
	@ApiOperation({
		description:
			'Get objects that cover the same subject as the passed object schema identifier. (elasticsearch: more like this)',
	})
```